test-cases/tests/monitoring/m05-verify-secrets.md

```diff
@@ -0,0 +1,47 @@
+---
+targets:
+  - 2.6.0
+---
+
+# M05 - Verify secrets
+
+More info:
+
+- <https://issues.redhat.com/browse/INTLY-4885>
+- <https://issues.redhat.com/browse/INTLY-9096>
+
+## Steps
+
+1. Login into OpenShift console as kubeadmin
+2. Navigate to `redhat-rhmi-middleware-monitoring-operator` namespace
+3. Open **Workloads > Secrets**
+4. Click on `alertmanager-application-monitoring` secret
+5. **Reveal Values**
+6. Make note of value for `receivers[1].pagerduty_configs.service_key`
+7. Navigate to `redhat-rhmi-operator` namespace
+8. Open **Workloads > Secrets**
+9. Click on `redhat-rhmi-pagerduty` secret
+10. **Reveal Values**
+    > Value for `serviceKey` key should match value noted earlier
+11. **Actions > Edit Secret**
+12. **Add Key/Value**
+13. For **Key** use `PAGERDUTY_KEY`
+14. For **Value** use `new_test`
+15. Check again `alertmanager-application-monitoring` secret in `redhat-rhmi-middleware-monitoring-operator` namespace
+    > Value for `receivers[1].pagerduty_configs.service_key` should be `new_test`
+16. Navigate to `redhat-rhmi-middleware-monitoring-operator` namespace
```

Comment:
@jhellar would it be worth adding sub headers for each secret? i.e ` ## Pagerduty, ## Dead Mans Snitch`. That way it would be easier to distinguish what secret you are testing. Also I'm not sure if additional secrets would be added in the future, but if so it would make it more readable.

Reply:
That makes sense, should be updated now.